datadog_checks_base/datadog_checks/base/stubs/datadog_agent.py

```diff
@@ -67,7 +67,7 @@ def read_persistent_cache(self, key):
 
     def obfuscate_sql(self, query):
         # This is only whitespace cleanup, NOT obfuscation. Full obfuscation implementation is in go code.
-        return re.sub(r'\s+', ' ', query or '')
+        return re.sub(r'\s+', ' ', query or '').strip()
```

Comment:
Let's https://datadoghq.dev/integrations-core/guidelines/pr/#separation-of-concerns

Reply:
Updated to mock the obfuscation in the test instead.